pysces/vortex.py

```diff
@@ -79,6 +79,9 @@ def induced_velocity_single(self, x, xvort, gam):
         gam : float
             Strength of vortex
 
+        This method returns a vector of velocities at the points x induced
```

Comment:
Use [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) conventions for the returned values.